src/Magnum/Trade/PhongMaterialData.h

```diff
@@ -299,11 +373,15 @@ class MAGNUM_TRADE_EXPORT PhongMaterialData: public AbstractMaterialData {
            and thus better noticeable */
         Color4 _ambientColor;
         UnsignedInt _ambientTexture{~UnsignedInt{}};
+        UnsignedInt _ambientCoordinateSet{UnsignedInt{}};
         Color4 _diffuseColor;
         UnsignedInt _diffuseTexture{~UnsignedInt{}};
+        UnsignedInt _diffuseCoordinateSet{UnsignedInt{}};
         Color4 _specularColor;
         UnsignedInt _specularTexture{~UnsignedInt{}};
+        UnsignedInt _specularCoordinateSet{UnsignedInt{}};
         UnsignedInt _normalTexture{~UnsignedInt{}};
+        UnsignedInt _normalCoordinateSet{UnsignedInt{}};
```

Comment:
```suggestion
        UnsignedInt _normalCoordinateSet;
```

Here it doesn't have to be set to anything anymore because it's initialized always (sorry)